### ISSUE TITLE
More multiplayer slot please

### ISSUE THREAD

**Issue by xdub** [diasurgical/devilutionX#543]
Hi, Could you add more slots for multiplayer players on next release? 4 players is really low, most games have at least 8, or better 16.
Thank you

**Comment by qndel** [comment]
have you even played with 4?

**Comment by xdub** [comment]
Yes, several times. But we already met at 6.

**Comment by AJenbo** [comment]
I plan to bump it to 9

**Comment by arrowgent** [comment]
will town portals be addessed with this update?

**Comment by AJenbo** [comment]
yeah thats required

**Comment by julealgon** [comment]
Is this a serious issue or are you guys just trolling? Honest question.

**Comment by yuripourre** [comment]
@julealgon It's serious. Having more players could also allow us to have mercenaries or maybe multiple Golems(summons) per player.

**Comment by AJenbo** [comment]
> Having more player slots could also allow us to have mercenaries or maybe multiple Golems(summons) per player. 

that issue should probably e addressed differently

**Comment by sskras** [comment]
[AJenbo](https://github.com/AJenbo) commented [on Jan 21, 2020](https://github.com/diasurgical/devilutionX/issues/543#issuecomment-576537267)

> I plan to bump it to 9

IIUC, the progress can be tracked in PR https://github.com/diasurgical/devilutionX/pull/6680.

**Comment by Tormentor667** [comment]
Out of curiosity, has the max cap increaed to 8 or 16?

**Comment by DakkJaniels** [comment]
> Out of curiosity, has the max cap increaed to 8 or 16?

Pretty sure it's still 4.

**Comment by qndel** [comment]
> Out of curiosity, has the max cap increaed to 8 or 16?

Why would it?

**Comment by Trihedraf** [comment]
I have ran a 6-player LAN game and there simply isn't enough XP, items, or gold in the game to go around with that many players. On top of that, outside of LAN the extra players add even more lag and make the experience worse. Also this isn't a bug or optional QoL change, it would be a massive balance change.

**Comment by arrowgent** [comment]
true, even with 4 players.  levelling up and getting items becomes a challenge.

modifying XP and drops could improve that when you add more players.


as mentioned, 4 players can lag pretty bad at times.  adding more would require improving the net code
at a minium: latency balacing to find a middle host


as awesome as it could be with a dedicated server hosting up to 128 players in a massive instance.  if 4-6 is a struggle, adding more wont be possible (yet)

**Comment by AJenbo** [comment]
And there isn't even servers :)
